cli/running/cluster_instance.go

```diff
@@ -124,6 +127,11 @@ func (inst *clusterInstance) Start() error {
 			return err
 		}
 		cmd.Env = append(cmd.Env, "TT_CONSOLE_SOCKET_DEFAULT="+consoleSocket)
+		_, listenSet := os.LookupEnv("TT_LISTEN")
+		_, listenDefaultSet := os.LookupEnv("TT_LISTEN_DEFAULT")
```

Comment:
There is no need to check this var. If a user wants to set listen, he/she just sets TT_LISTEN. It is not overwritten by TT_LISTEN_DEFAULT.

Reply:
changed